.github/workflows/bff-ci.yml

```diff
@@ -47,13 +47,12 @@ jobs:
       run: dotnet build bff.sln -c Release
     - name: Dotnet devcerts
       run: dotnet dev-certs https --trust
+    - name: Install Playwright
+      run: pwsh test/Hosts.Tests/bin/Release/net9.0/playwright.ps1 install --with-deps
```

Comment:
This is such [a horrible way to install](https://github.com/microsoft/playwright-dotnet/issues/2286) :(

Reply:
I totally agree. In the future I may also need nodejs for some scneairos.. there's a cleaner nodejs based install but I don't know if it works for the .net version of playwright.